MyOnboarding/src/MyOnboardingApp.Contracts/Models/TodoListItem.cs

```diff
@@ -8,6 +8,15 @@ public class TodoListItem
 
         public string Text { get; set; }
 
-        public override string ToString() => $"{nameof(Id)}: {Id}, {nameof(Text)}: {Text}";
+        public DateTime CreationTime { get; set; }
+
+        public DateTime LastUpdateTime { get; set; }
+
+
+        public override string ToString()
+            => $"{nameof(Id)}: {Id}, " +
+               $"{nameof(Text)}: {Text}, " +
+               $"{nameof(CreationTime)}: {CreationTime:0:MM/dd/yy H:mm:ss}" +
+               $"{nameof(LastUpdateTime)}: {LastUpdateTime:0:MM/dd/yy H:mm:ss}";
```

Comment:
```suggestion
               $"{nameof(LastUpdateTime)}: {LastUpdateTime:s}";
```
This is a bit shorter and, what seems most important to me: it produces "ordered" (sortable) result that is easier to read. I would have nothing against something like `yyyy-MM-dd HH:mm:ss` either, but since you use it twice and it is a custom-ish date format, I would preferably see that as a constant… The fact that `s` should effectively also be hidden behind a constant is something I'm willing to overlook 😆